.cirrus.yml

```
@@ -0,0 +1,84 @@
+env:
+  ELIXIR_ASSERT_TIMEOUT: 2000
+  LANG: C.UTF-8
+
+# test_task:
+#   container:
+#     image: ubuntu:14.04
+#
+#   matrix:
+#     - env:
+#         OTP_RELEASE: OTP-21.3
+#         CHECK_REPRODUCIBLE: true
+#         CHECK_POSIX_COMPLIANT: true
+#     # - env:
+#     #     OTP_RELEASE: OTP-21.2
+#     # - env:
+#     #     OTP_RELEASE: OTP-21.1
+#     # - env:
+#     #     OTP_RELEASE: OTP-21.0
+#     # - env:
+#     #     OTP_RELEASE: OTP-20.3
+#     # - env:
+#     #     OTP_RELEASE: OTP-20.2
+#     # - env:
+#     #     OTP_RELEASE: OTP-20.1
+#     - env:
+#         OTP_RELEASE: OTP-20.0
+#     # - env:
+#     #     OTP_RELEASE: maint
+#     #   allow_failures: true
+#     # - env:
+#     #     OTP_RELEASE: master
+#     #   allow_failures: true
+#
+#   install_script:
+#     - apt-get update
+#     - apt-get install -y wget build-essential git shellcheck
+#     - wget -O otp.tar.gz https://repo.hex.pm/builds/otp/ubuntu-14.04/${OTP_RELEASE}.tar.gz
+#     - mkdir -p otp
+#     - tar zxf otp.tar.gz -C otp --strip-components=1
+#     - otp/Install -minimal $(pwd)/otp
+#
+#   test_script:
+#     - PATH=$(pwd)/otp/bin:$PATH
+#     - rm -rf .git
+#     - ELIXIRC_OPTS="--warnings-as-errors" ERLC_OPTS="+warning_as_errors" make compile
+#     - make test
+#     - dialyzer -pa lib/elixir/ebin --build_plt --output_plt elixir.plt --apps lib/elixir/ebin/elixir.beam lib/elixir/ebin/Elixir.Kernel.beam
+#
+#     # Check for reproducible builds only in the latest OTP release
+#     - if [ -n "$CHECK_REPRODUCIBLE" ]; then make check_reproducible; fi
+#
+#     # Check for POSIX compliant shell scripts
+#     - if [ -n "$CHECK_POSIX_COMPLIANT" ]; then
+#         shellcheck -e SC2039,2086 bin/elixir && echo "bin/elixir is POSIX compliant";
+#         shellcheck bin/elixirc && echo "bin/elixirc is POSIX compliant";
+#         shellcheck bin/iex && echo "bin/iex is POSIX compliant";
+#       fi
+
+test_windows_task:
+  windows_container:
+    image: cirrusci/windowsservercore:2019
+    os_version: 2019
+
+  env:
+    CIRRUS_SHELL: powershell
+
+  matrix:
+    # - env:
+    #     OTP_RELEASE: 20.0
+    - env:
+        OTP_RELEASE: 21.0
+    # - env:
+    #     OTP_RELEASE: 22.0
+
+  install_script:
+    - choco install -y erlang --version !OTP_RELEASE!
```

I think it should be like this:

```suggestion
    - choco install -y erlang --version $Env:OTP_RELEASE
```

---

BTW I enjoyed your commit messages. 🤣 Windows is a pains. 😪

---

😂 

Thanks!